pkg/webui/account/components/oauth-client-form/messages.js

```diff
@@ -54,4 +54,11 @@ export default defineMessages({
   grantTypeAndRights: 'Grant types and rights',
   stateDescriptionDesc:
     'You can use this field to save additional information about the state of this OAuth client, e.g. why it has been flagged',
+  contactWarning:
+    'Note that if no contact is provided, it will default to the first collaborator of the client.',
```

Comment:
It's going to be removed anyways but for the record: there's no point in redefining this same message over and over for each entity. This is what we have shared messages for.